README-ruRU.md

```diff
@@ -3682,13 +3682,13 @@
   end
 
   linux_organization = Organization.find(...)
-  # BAD - violates privacy
+  # ПЛОХО - нарушает private области видимости
   linux_organization.send(:reset_token)
```

Comment:
Please keep good/bad lowercased throughout the whole guide. The capitalization at this point in the English version is a BUG. Feel free to submit a patch to Bozhidar's version.